tuning-crc-openshift-cluster/trigger-podpresets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+curdir="$(dirname $0)"
+for namespace in $(oc get ns  -ojsonpath='{.items[*].metadata.name}')
+do
+	if [[ "$namespace" =~ "openshift-" ]]; then
+			success=0
+			sed 's/NAMESPACE_TO_REPLACE/"'"$namespace"'"/g' "$curdir"/podpreset-template.yaml > trigger-podpreset.yaml
+			echo 'creating podpreset for the namespace: "'"$namespace"'"' 
+			for i in {1..10}; do
```

Comment:
This loop {1..10} is not required. Podpresets not getting enabled at 'enable-alphaapi.sh' can be the reason for failure to create podpresets (no matter if we have loop here or not)